Makefile

```diff
@@ -26,19 +42,73 @@ export DOCKER_BUILDKIT:=1
 help:
 	@echo "jupyter/docker-stacks"
 	@echo "====================="
-	@echo "Replace % with a stack directory name (e.g., make build/minimal-notebook)"
+	@echo "Replace % with a stack directory name (e.g., make build-multi/minimal-notebook)"
 	@echo
 	@grep -E '^[a-zA-Z0-9_%/-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 
 
 build/%: DARGS?=
-build/%: ## build the latest image for a stack
-	docker build $(DARGS) --rm --force-rm -t $(OWNER)/$(notdir $@):latest --build-arg OWNER=$(OWNER) ./$(notdir $@)
+build/%: ## build the latest image for a stack using the system's architecture
+	@echo "::group::Build $(OWNER)/$(notdir $@) (system's architecture)"
+	docker build $(DARGS) --rm --force-rm -t $(OWNER)/$(notdir $@):latest ./$(notdir $@) --build-arg OWNER=$(OWNER)
```

Comment:
I will create a separate PR for this as well, because it's not actually a part of arm support.

Reply:
I don't think that's easy though :(

Reply:
Seems to be easy :)